index.html

```diff
@@ -0,0 +1,198 @@
+      <h2 class="item">My Recent Works</h2>
+      <hr/>
+    </section>
+    <section class="card grid">
+      <div class="item img-holder grid-item-1">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-2">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-3">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li  class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-4">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+            </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-5">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-6">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
```

Comment:
- Kindly, when the previous code reviewer suggested using `hx` means `h1`-`h6` :+1: :+1:  update it because `hx` unknown tag